stories/documentation/texts/code/angular/code.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { codeComponent } from '@lucca-front/ng/code';
+import { Meta, moduleMetadata } from '@storybook/angular';
+import { cleanupTemplate } from 'stories/helpers/stories';
+
+interface CodeBasicStory {
+	block: boolean;
+	content: string;
+}
+
+export default {
+	title: 'Documentation/Texts/Code/Angular/Basic',
+	argTypes: {
+		block: {
+			type: 'boolean',
+		},
+		content: {
+			control: {
+				type: 'text',
+			},
+		},
+	},
+	decorators: [
+		moduleMetadata({
+			imports: [codeComponent],
+		}),
+	],
+	render: (args: CodeBasicStory) => {
+		const blockParam = args.block ? ` block` : ``;
+		return {
+			template: cleanupTemplate(`<lu-code${blockParam}>${args?.content}</lu-code$>`),
```

Comment:
There is a compilation error, that might be related to the `$` at the end of the line.

Reply:
<img width="1050" height="702" alt="image" src="https://github.com/user-attachments/assets/ca9117b3-371d-49af-8fa3-93880a916ac8" />